Source/init.cpp

```diff
@@ -143,6 +144,12 @@ void init_archives()
 	if (paths[0] == paths[1])
 		paths.pop_back();
 
+	char gogpath[_FSG_PATH_MAX];
+	fsg_get_gog_game_path(gogpath, "1412601690");
+	if (strlen(gogpath) > 0) {
+		paths.emplace_back(std::string(gogpath) + "/");
+		paths.emplace_back(std::string(gogpath) + "/hellfire/");
+	}

```

Comment:
Despite this being a cross-platform lib the current version only supports GOG on Windows, and Diablo will probably never be available under other platforms either, so preferably this should be behind `#if defined(_WIN64) || defined(_WIN32)` (same with the include)

Reply:
I think this is fine because on other platforms, the library will just return an empty string and no paths will be added

Reply:
Let's not link and run extra code on platforms where it won't do anything

Reply:
fair enough